iosMath/render/MTMathListDisplay.m

```diff
@@ -419,11 +468,19 @@ - (void)draw:(CGContextRef)context
     CGFloat heightFromTop = _topKern;
 
     // draw the horizontal line with the given thickness
+#if TARGET_OS_IPHONE
     UIBezierPath* path = [UIBezierPath bezierPath];
+#else
+    NSBezierPath* path = [NSBezierPath bezierPath];
```

Comment:
Do this one same as `MTColor` as well
And then add a category on `NSBezierPath` which adds the method `addLineToPoint`, then we can get rid of all these `#if #else`